docs/setup/import.md

```diff
@@ -48,18 +48,62 @@ Once the import is completed, you can merge the changes from the import branch t
 
 ### Importing using lakectl cli
 
-The `lakectl` command supports ingesting objects from an external source.
-This is done by listing the source bucket (and optional prefix), and creating pointers to the returned objects in lakeFS.
+The `lakectl` cli supports _import_ and _ingest_ commands to import objects from an external source.
 
-#### Prerequisites
+- The _import_ command uses the same way as the UI import the lakeFS access to import (zero copy) and commit the changes on `_<branch_name>_imported` branch with optional flag to also merge the changes to `<branch_name>`.
+- The _Ingest_ is done by listing the source bucket (and optional prefix) from the client, and creating pointers to the returned objects in lakeFS. The objects will be staged on the branch.
 
-1. The user calling `lakectl ingest` has permissions to list the objects at the source object store.
-2. _recommended_: The lakeFS installation has read permissions to the objects being ingested (to support downloading them directly from the lakeFS server)
-3. The source path is **not** a storage namespace used by lakeFS. For example, if `lakefs://my-repo` created with storage namespace `s3://my-bucket`, then `s3://my-bucket/*` cannot be an ingestion source.
+
+#### Using the `lakectl import` command
+
+##### Usage
+
+<div class="tabs">
+<ul>
+  <li><a href="#import-tabs-1">AWS S3 or S3 API Compatible storage</a></li>
+  <li><a href="#import-tabs-2">Azure Blob</a></li>
+  <li><a href="#import-tabs-3">Google Cloud Storage</a></li>
+</ul>
+<div markdown="1" id="import-tabs-1">
+```shell
+lakectl import \
+  --from s3://bucket/optional/prefix/ \
+  --to lakefs://my-repo/my-branch/optional/path/
+```
+</div>
+<div markdown="1" id="import-tabs-2">
+```shell
+lakectl import \
+   --from https://storageAccountName.blob.core.windows.net/container/optional/prefix/ \
+   --to lakefs://my-repo/my-branch/optional/path/
+```
+</div>
+<div markdown="1" id="import-tabs-3">
+```shell
+lakectl import \
+   --from gs://bucket/optional/prefix/ \
+   --to lakefs://my-repo/my-branch/optional/path/
+```
+</div>
+</div>
+
+Note that `lakectl import` will create or use `_<my-branch>_imported` branch (name is based on my-branch name).
+The objects will be imported and commit to this branch.
+
+
+Passing `--merge` flag will perform a merge from `_<my-branch>_imported` branch to `<my-branch>` branch, after a successful import.
```

Comment:
```suggestion
The imported objects will be committed to `_my-branch_imported` branch. If the branch does not exist, it will be created. The flag `--merge` will merge the branch `_my-branch_imported` to  `my-branch` after a successful import.
```